okteto/okteto
# DEV-624: Added scan for okteto/bin image, and update okteto/bin dependency to 1.6.1

# Proposed changes

Fixes DEV-624 and DEV-623

In this PR I'm updating the version of `okteto/bin` to `1.6.1` (https://hub.docker.com/layers/okteto/bin/1.6.1/images/sha256-2a65334a52baf1d803156be49e935f4bc69f34d6d424a19015f6050f32e03946?context=explore) to fix a vulnerability on one of its dependencies. I'm also adding a new image scan when a CLI image is pushed to scan `okteto/bin:latest` image

Also, I removed the file `release-bin.yml` as it was a leftover from when we moved the okteto/bin image to another repository

## How to validate

1. Create a binary from this branch
2. Execute `okteto up` command on any dev environment
3. Review the dev pod spec in Kubernetes to ensure that `okteto/bin:1.6.1` is being used

```
  initContainers:
  - command:
    - sh
    - -c
    - cp /usr/local/bin/* /okteto/bin
    env:
    - name: OKTETO_DOMAIN
      value: xxxxx
    - name: OKTETO_NAMESPACE
      valueFrom:
        fieldRef:
          apiVersion: v1
          fieldPath: metadata.namespace
    image: okteto/bin:1.6.1
    imagePullPolicy: IfNotPresent
```

This is the output of one of my tests
